### PR TITLE
refactor: use simpler interfaces

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/VoteProxy.sol [hunk 1]
```diff
@@ -18,18 +18,32 @@
...
+
+interface ChiefLike {
+    function GOV() external view returns (TokenLike);
+    function IOU() external view returns (TokenLike);
+    function deposits(address) external view returns (uint256);
+    function lock(uint256) external;
+    function free(uint256) external;
+    function vote(address[] calldata) external returns (bytes32);
+    function vote(bytes32) external;
+}
```
Comment (godsflaw) [inline src/VoteProxy.sol:36]: Alternative here would be to pull in `dss-interfaces`.

**Review by brianmcmichael — APPROVED** [review]
lgtm

**Review by godsflaw — APPROVED** [review]
LGTM

File: src/VoteProxy.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/VoteProxy.sol:36]: So far we have been using simpler interfaces (with the minimum) for the actual contracts. But yeah this is probably something to discuss more deeply.

**Pull request merged**